build.sbt

```diff
@@ -31,13 +31,23 @@ lazy val munit: Seq[Setting[_]] = Seq(
 )
 
 Compile / doc / scalacOptions ++= Seq(
-  "-d",
-  "generated-docs",
-  "-project",
-  "Scala-yaml",
-  "-siteroot",
-  "docs"
-)
+  "-project", "Scala-yaml",
+  "-siteroot", "docs",
+  "-project-version", version.value,
+  "-project-logo", "docs/logo.svg",
+  "-social-links:" +
+    "github::https://github.com/VirtusLab/scala-yaml," +
+    "twitter::https://twitter.com/VirtusLab",
+  "-project-footer", s"Copyright (c) 2021, VirtusLab",
+  "-source-links:github://VirtusLab/scala-yaml",
+  "-revision", "master"
```

Comment:
According to [docs](https://dotty.epfl.ch/docs/usage/scaladoc/settings.html) `-siteroot docs` and `-revision master` are default options so they can be omitted.

Reply:
Docs are misleading since we need them - we will work on scaladoc docs